src/Layer/LabelLayer.js

```diff
@@ -44,7 +44,9 @@ class LabelLayer extends Layer {
             this.domElement.style.display = this.visible ? 'block' : 'none';
         });
 
-        this.parsingOptions.buildExtent = true;
+        // Feature options ??
+        // move to layer...because generaly the Source is shared
```

Comment:
Comment ?

Reply:
In `View`, I add `labelLayer.buildExtent = true` when it's created by the option `layer.enableLayer` 
Because the features are shared between this `layer` and `labelLayer`.